package.json

```diff
@@ -44,21 +46,25 @@
   },
   "homepage": "https://github.com/livepeer/protocol#readme",
   "devDependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.10",
```

Comment:
I'm guessing we need something from this particular version? If so, what do we need?

Reply:
the hardhat version of ethers has some slight modifications that can be found here https://hardhat.org/plugins/nomiclabs-hardhat-ethers.html#helpers

hardhat-deploy further extends this functionality with helpers such as getting a signer based on an address and some other functions , you can view the full interface extension here: https://www.npmjs.com/package/hardhat-deploy-ethers

This version was automatically installed when installing `hardhat-deploy`